log10/feedback/feedback.py

```diff
@@ -111,3 +160,40 @@ def _get_feedback_list(offset, limit, task_id):
         return []
 
     return feedback_data
+
+
+def _format_graphql_node(node):
+    return {
+        "id": node["id"],
+        "json_values": node["jsonValues"],
+        "task_id": node["task"]["id"],
+        "task_name": node["task"]["name"],
+        "matched_completion_ids": [c["id"] for c in node["completions"]],
+    }
+
+
+def _get_feedback_list_graphql(page, limit, task_id, filter):
+    feedback_data = []
+    current_page = page if page else 1
+    if limit:
+        limit = int(limit)
+
+    try:
+        while True:
+            fetch_limit = limit if limit else 50
```

Comment:
This can be written as a default arg eg
`def _get_feedback_list_graphql(page, task_id, filter, limit=50):`